css/style.css

```
@@ -1168,50 +1168,46 @@ p {
     gap: 30px;
 }
 
-.playlist__img {
-    /* position: relative; */
-}
-
 .playlist__card {
-    /* position: relative; */
     cursor: pointer;
 }
 
 .playlist__card-wrapper {
     position: relative;
+    background: var(--color-black);
 }
 
-
-/* .playlist-btn {
-
+.playlist__btn {
+    display: none;
     position: absolute;
-    left: 50%;
-    top: 50%;
-    display: block;
-    width: 48px;
-    height: 48px;
-    background-image: url(../img/playlist-play.svg);
-    background-repeat: no-repeat;
-    background-position: center center;
-
-} */
+    top: 0;
+    left: 0;
+    z-index: 3;
+    top: calc(50% - 21px);
+    left: calc(50% - 21px);
```

Не очень классно использовать конкретные значения в пикселях, когда тебе нужно что-то абсолютно спозиционировать по центру
```suggestion
    left: 50%;
    top: 50%;
    transform: translate(-50%, -50%0;
```